data/json/species.json

```diff
@@ -68,15 +69,18 @@
   {
     "type": "SPECIES",
     "id": "WORM",
+    "footsteps": "rustle.",
```

Comment:
Translatable strings in JSON require separate code in "lang/extract_json_strings.py" to extract them for translation.

Reply:
I've updated Python script - hope everything was correct.

CC: @BrettDong